Status.sh

```diff
@@ -152,6 +152,38 @@ then
   freedns_id_pass="\Zb\Z5FreeDNS ID and pass\Zn"
 fi
 
+# Verify API_SECRET since it can be edited after installation
+apisec_problem="" # No flag
+apisec_literal=$(grep 'API_SECRET=' /etc/nsconfig) # Extract the line containing API_SECRET= from the nsconfig file (including the quotation marks).
+apisec_literal=$(echo "$apisec_literal" | sed 's/^.*=//') # Drop everything up to and including the equal sign.
+if [[ "$apisec_literal" == *" #"* ]] # Is there a comment?
+then
+  apisec_literal=${apisec_literal%%#*} # Remove the comment and everything after.
+fi
+apisec_literal=$(echo "$apisec_literal" | awk '{$1=$1};1') # Remove trailing spaces
+apisec_literal="$apisec_literal"
+first="${apisec_literal:0:1}" # The first character, which should be either ' or "
+last="${apisec_literal: -1}" # The last character, which should be either ' or "
+if [[ "$first" == "$last" ]] # Are the first and last characters identical?
+then
+  if [[ "$first" == "'" ]] || [[ "$first" == "\"" ]] # Is the first character either ' or "
+  then
+    apisec_literal="${apisec_literal:1: -1}" # Remove the first and last characters (quotation mark pair)
+    if [[ "$apisec_literal" == *"@"* ]] || [[ "$apisec_literal" == *" "* ]] || [[ "$apisec_literal" == *"/"* ]] || [[ "$apisec_literal" == *"\\"* ]] || [[ "$apisec_literal" == *"'"* ]] || [[ "$apisec_literal" == *"\""* ]] || [[ "$apisec_literal" == *"$"* ]] || [[ ${#apisec_literal} -lt 12 ]] # Is an illegal character present?
+    then
+      apisec_problem="*" # Mark the presence of one of the seven illegal characters.  
```

Comment:
Consider marking each of them with a different char or a number (for example *1, *2 and so on). This will make it easier for you to understand what the error is.

Reply:
I was going to do that.  But then, I changed my mind.

We agreed that this may not be a catastrophic error.  It depends on the uploader if it is an error or not.  At least, not all of the characters.
So, we want to allow people to keep using the setup even if they have a character that does not meet all the requirements.

If we do that, we should not be revealing the characters users are using in their API_SECRET.  And my intent is to put a clear description of what the flag means in the guide.  So, if we put a list showing exactly what character the user has may not be a good idea.
That's why I changed my mind.